playbooks/openshift-hosted/config.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - import_playbook: ../init/main.yml
   vars:
-    l_init_fact_hosts: "oo_masters_to_config"
+    l_init_fact_hosts: "oo_masters_to_config:glusterfs_registry:glusterfs"
```

Comment:
@jarrpa are there any corresponding oo_ groups for these gluster hosts, or we don't have any for those?  I know in some other files we use oo_gluster_to_config, which would be helpful if we can stay consistent.

Reply:
Sorry for the delay, this fell off my radar. Totally right, updating.